locale/en/download/package-manager.md

```diff
@@ -356,6 +356,14 @@ Download the [Windows Installer](https://nodejs.org/en/#home-downloadhead) direc
 
 ### Alternatives
 
+Using **[Winget](https://aka.ms/winget-cli)**:
+
+```bash
```

Comment:
```suggestion
```powershell
```

I wonder if this should be `powershell` for the info strings?

Reply:
> I wonder if this should be `powershell` for the info strings?

I tested this (simply running `winget --version`) in Git Bash and it did output its version, so I think the most appropriate info string for this one would be `console`, which is shell-agnostic.

```suggestion
```console
```